src/Microsoft.Data.Sqlite/SqliteProviderFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Data.Common;
+
+namespace Microsoft.Data.Sqlite
+{
+    public class SqliteProviderFactory : DbProviderFactory
```

Comment:
Make this a singleton with a private constructor and a static `Instance` property.